colcon_package_information/verb/list.py

```diff
@@ -120,10 +120,12 @@ def main(self, *, context):  # noqa: D102
                     if j == i:
                         # package i is being processed
                         lines[j] += '+'
-                    elif shown_decorators[j].descriptor.name in depends[i]:
+                    elif shown_decorators[j].descriptor.name \
+                            in set(map(lambda d: d.name, depends[i])):
```

Comment:
Can you rewrite this condition to:
```
elif (
    shown_decorators[j].descriptor.name in
    {d.name for d in depends[i]}
):
```

(avoiding the double indentation as well as the nested set/map/lamda).

Same below.